src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.props

```diff
@@ -148,7 +148,7 @@
   <Target Name="CreateDotVersionFile"
           DependsOnTargets="InitializeSourceControlInformationFromSourceControlManager"
           BeforeTargets="GetFilesToPublish"
-          Condition="'$(DisableSourceLink)' != 'true'">
+          Condition="'$(EnableSourceLink)' != 'false'">
```

Comment:
I see other instances of `DisableSourceLink` should they all be removed?  https://github.com/dotnet/runtime/search?q=DisableSourceLink

Reply:
I think these are correct. It seems to me that DisableSourceLink is an opt-out switch, which causes Arcade to set EnableSourceLink (which is a bit weird on the design side). So you can use it to disable source link.

But, in places in this repo where we need to influence other behavior based on whether sourcelink is on, it should be using the EnableSourceLink switch.

Reply:
I think this one should really use `EnableSourceControlManagerQueries`